src/Geometry.cpp

```diff
@@ -163,3 +250,19 @@ void Geometry::Destroy() const {
     glDeleteBuffers( 1, &colourBuffer );
     glDeleteBuffers( 1, &normalBuffer );
 }
+
+void Geometry::GetCorkTriMesh( CorkTriMesh& out ) {
+    // Dynamically create copies of geometry data
+    std::vector<GLuint>* nfaces = new std::vector<GLuint>( faces );
+    std::vector<glm::vec3>* nvertices = new std::vector<glm::vec3>( vertices );
+
+    for ( auto& v : *nvertices ) {
```

Comment:
??? is this doing anything

Reply:
Transforming the geometry::vertices to world space